monitoring/prometheus.md

```diff
@@ -42,7 +42,7 @@ oc process -f fuse-servicemonitor.yml -p NAMESPACE=<your-fuse-namespace> FUSE_SE
 
 Once you have done this, you should be able to view metrics for your Fuse application in the Monitoring->Metrics section.     Type "org_apache_camel_ExchangesTotal" or another camel metric into the query box :
 
-![Exchanges Total](https://github.com/jboss-fuse/application-templates/raw/master/prometheus/exchangestotal.png)
+![Exchanges Total](https://github.com/jboss-fuse/application-templates/raw/master/monitoring/exchangestotal.png)
```

Comment:
For the newly added png files, you can utilise the `img/` directory as well, so that the monitoring directory doesn't get cluttered :-)